src/test/java/spec/concordion/ExampleTest.java

```diff
@@ -6,7 +6,7 @@
 
 import test.concordion.TestRig;
 
-@RunWith(ConcordionRunner.class)
+@RunWith(ConcordionRunner.class)

```

Comment:
Please remove all the files with this change from the commit, since they add too much noise